docs/walkthrough/loading-models.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 1
+---
+
+# Loading Models
+
+<!-- THE MARKDOWN (.md) FILE IS GENERATED FROM THE NOTEBOOK (.ipynb) FILE -->
+
+Substratus makes it very easy to import your existing ML models into Substratus.
+You can for example use the Substratus provided `substratusai/model-loader-huggingface`
+container image to load models into Substratus.
+
+The below Model resource shows an example on how to load the falcon-7b-instruct
+model into substratus:
+```yaml
+apiVersion: substratus.ai/v1
+kind: Model
+metadata:
+  name: falcon-7b-instruct
+spec:
+  image:
+    name: substratusai/model-loader-huggingface
+  params:
+    name: tiiuae/falcon-7b-instruct
+```
+
+In the model spec, you provide the container image location and the name
+of the HuggingFace model. The model-loader-huggingface downloades the model
+and stores it in a predefined location such that the model gets stored
+in a cloud storage bucket (e.g. a GCS bucket).
+
+The Model.status.url will provide you with the cloud storage location
```

Comment:
To stay consistent with other docs: "The Model `.status.url`"

Reply:
fixed